wallet/how-to/use-non-evm-networks/starknet/index.md

```diff
@@ -0,0 +1,98 @@
+---
+description: Interact with users' Starknet accounts in MetaMask.
+sidebar_position: 1
+---
+
+# Use Starknet
+
+Starknet is a Layer 2 network built on Ethereum. To interact with Starknet accounts in MetaMask, which is a non-EVM network, you need to use the Starknet Snap.
+
+You can use `wallet_invokeSnap` or `get-starknet` to integrate with Starknet dapps.
+The choice depends on your specific use case and development preferences. 
+
+Both methods enable developers to connect Starknet dapps with wallet solutions. 
+
+`wallet_invokeSnap`:
+- Requires precise method names and parameter structures.
+- Handles both MetaMask-specific and Starknet-specific errors.
+- Designed for operating within the MetaMask framework.
+- Manages lower-level StarkNet interactions directly.
+- Results are in more detailed, lower-level code.
```

Comment:
```suggestion
`wallet_invokeSnap`:
- Requires precise method names and parameter structures.
- Handles both MetaMask-specific and Starknet-specific errors.
- Is designed for operating within the MetaMask framework.
- Manages lower-level Starknet interactions directly.
- Provides results in more detailed, lower-level code.
```

This might require further edits